tcpMaxConn.go

```diff
@@ -28,16 +32,70 @@ func runTcpConnectionsInParallel(numberConnections int, delay int, host string,
 	wg.Wait()
 }
 
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+func askForUserConfirmation(assumeyes bool, host string, port string, connections int) bool {
+	var executeBool = false
+	if assumeyes == false {
+		fmt.Println("*** WARNING ***")
+		fmt.Println("You are going to run a TCP stress check with the following arguments:")
+		fmt.Println("  - Target: " + host)
+		fmt.Println("  - TCP Port: " + port)
+		fmt.Println("  - # of concurrent connections: " + strconv.Itoa(connections))
+		reader := bufio.NewReader(os.Stdin)
+		for {
+			fmt.Println("Please, would you confirm this execution? (y/N)")
+			response, err := reader.ReadString('\n')
+			if err != nil {
+				fmt.Println("Response not processed")
+				os.Exit(1)
+			}
+			response = strings.TrimSuffix(response, "\n")
```

Comment:
if you also use https://golang.org/pkg/strings/#ToLower, you save some extra comparisons..